iam/terraform/account-specific/main/circle-ci.tf

```diff
@@ -70,22 +70,22 @@ resource "aws_iam_policy" "circle_ci_policy" {
         "cognito-idp:UpdateUserPool",
         "cognito-idp:CreateUserPool",
         "cognito-idp:AdminRespondToAuthChallenge",
-        "cognito-idp:AdminConfirmSignUp",
         "cognito-idp:DescribeUserPool",
         "cognito-idp:CreateUserPoolDomain",
         "cognito-idp:UpdateUserPool",
         "cognito-idp:DescribeUserPoolClient",
         "cognito-idp:AdminInitiateAuth",
         "cognito-idp:AdminDisableUser",
-        "cognito-idp:SignUp",
         "cognito-idp:ListUserPools",
         "cognito-idp:ListUserPoolClients",
         "cognito-idp:CreateUserPoolClient",
         "cognito-idp:DescribeUserPoolDomain",
         "cognito-idp:SetUICustomization",
         "cognito-idp:DeleteUserPoolDomain",
         "cognito-idp:GetUserPoolMfaConfig",
-        "cognito-idp:SetUserPoolMfaConfig"
+        "cognito-idp:SetUserPoolMfaConfig",
```

Comment:
Don't we need to separately run the `account-specific` deploy to apply these policy updates, or is this somehow covered in the CircleCI deployment?

Reply:
These changes won't apply to existing environments, only new environments.

Reply:
That's a good point Julia. It's something we ran into when rebuilding Dev. It's a step we ought to have to perform for all environments Thank you for flagging.